modules/openapi-generator-core/src/main/java/org/openapitools/codegen/config/GeneratorSettings.java

```diff
@@ -52,7 +52,7 @@ public final class GeneratorSettings implements Serializable {
     private final Map<String, String> importMappings;
     private final Map<String, String> inlineSchemaNameMappings;
     private final Set<String> languageSpecificPrimitives;
-    private final Map<String, String> reservedWordMappings;
+    private final Map<String, String> reservedWordsMappings;
```

Comment:
I have a feeling this might break other PRs that are in progress and use this variable or getter

Reply:
Well that needs to be solved in whatever PR is merged last :)

Reply:
this is a change in public API of openapi-generator. this will also break external forks of openapi generators that a lot of companies manage on their own, it's not really dart related

Reply:
Not sure. According to the Docs, this is what it should be named like and what the CLI arg is named like.
This just brings the parser for YAML config in line.
See https://openapi-generator.tech/docs/usage/

Reply:
Imho this is a bug and not matching the docs.

Reply:
What about updating the doc instead?

Reply:
@wing328 The `[--reserved-words-mappings <reserved word mappings>...]` works correct for CLI args, just not for YAML-file configuration because of the typo. There is no separate documentation for the config file, so I think it should be fixed to match.

Reply:
ok. let's go with this fix. Thanks for the PR as always